### PR TITLE
Reuse copy-bookmarklet script

### PR DESCRIPTION
**Pull request by johnholdun**
This change moves the inline script used for copying text to the clipboard into a separate file and adds it to the page as necessary. It also refactors that script very slightly, and fixes one use of the script that was not working properly—seems like maybe the code got reformatted as if it was a block of plain text, which broke some of the code.

### CODE REVIEW

**Review by ckolderup — CHANGES_REQUESTED** [review]
This isn't your fault, you were just copying off of my incorrect nomenclature, but could you do me a favor and make a couple small changes while you're in here?

* rename the script to `copy-to-clipboard.js` (or similar)
* change from using an ID to using a similarly-named class on elements that use it
* change the javascript to look for that class

it's bugging me that I repurposed the field for copying the Mastodon-compatible username and didn't rename it.

Thanks! Sorry!

**Comment by johnholdun** [comment]
Heck yeah I can

**Comment by johnholdun** [comment]
Done! I made this powered by data-* attributes, and also changed the two inputs to buttons so they're more easily discoverable as interactive elements.

Because buttons overflow text differently than text inputs do, I also added a hardcoded display-only version of the bookmarklet javascript that is added to the button for display purposes. It might ultimately be better in this case if this button just said "Copy to clipboard" and was its own status indicator (which I think would work without any other modifications—the data-copy-status selector can just point to itself)

File: src/pages/admin/index.hbs [hunk 1]
```diff
@@ -6,16 +6,15 @@
   new bookmark in your browser's bookmarks toolbar, then edit the URL for that
   bookmark and insert this javascript:
 </p>
-<input readonly type="text" id="copy-bookmarklet" value="{{{bookmarklet}}}" />
-<p><small id="copy-status">Click the code above to copy</small></p>
+<button
+  class="btn"
+  data-copy="{{{bookmarklet}}}"
+  data-copy-status=".copy-bookmarklet-status-status"
```
Comment (ckolderup) [inline src/pages/admin/index.hbs:12]:
```suggestion
  data-copy-status=".copy-bookmarklet-status"
```

I think there was a find & replace problem here!

**Comment by ckolderup** [comment]
oh wow I *can* commit changes to the PR, at least via Github's suggestion feature 🤔

**Review by ckolderup — APPROVED** [review]
manually tested and this is all working nicely! Thanks @johnholdun and congrats on your first contribution 🥲

**Pull request merged**